protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -53,6 +53,41 @@ In the example below there are 3 liquidity providers all bidding for their chose
 1. If the `target stake = 240` then even putting all the liquidity supplied above does not meet the estimated market liquidity demand and thus we set `k=N` and so the market's liquidity-fee-factor is `LP N fee = LP 3 fee = 3.75%`.
 1. Initially (before market opened) the `[target stake]` is by definition zero (it's not possible to have a position on a market that's not opened yet). Hence by default the market's initial liquidity-fee-factor is the lowest liquidity-fee-factor.
 
+
+### Stake-weighted-average method for setting the liquidity fee factor
+
+The liquidity fee factor is set as the weighted average of the liquidity fee factors, with weights assigned based on the supplied stake from each liquidity provider, which can also account for the impact of one supplier's actions on others.
+
+#### Example for fee setting mechanism using the Stake-weighted-average method
+
+In the example below there are 3 liquidity providers all bidding for their chosen fee level. The overall liquidity fee factor is the weight-average of their nominations:
+
+```text
+[LP 1 stake = 120 ETH, LP 1 liquidity-fee-factor = 0.5%]
+[LP 2 stake = 20 ETH, LP 2 liquidity-fee-factor = 0.75%]
+[LP 3 stake = 60 ETH, LP 3 liquidity-fee-factor = 3.75%]
+
+then
+
+liquidity-fee-factor = (120 * 0.05) + (20 * 0.075) + (60 8 0.0375) / (120 + 20 + 160) = 0.04875
```

Comment:
```suggestion
liquidity-fee-factor = ((120 * 0.5%) + (20 * 0.75%) + (60 * 3.75%)) / (120 + 20 + 160) = 1.5%
```

Reply:
I'm probably doing something wrong, but that now works out as 1% not 1.5%?